spec/features/user_adds_coverage_to_a_course_spec.rb

```diff
@@ -1,21 +1,27 @@
 require "rails_helper"
 
 feature "user adds coverage to a course" do
-  scenario "successfully" do
+  scenario "successfully", js: true do
     subject = create(:subject)
     course = create(:course)
     outcome = create(:outcome, course: course)
     user = user_with_assessments_access_to(course.department)
 
     visit manage_assessments_course_path(course, as: user)
     click_on t('manage_assessments.courses.show.add_a_class')
-    select subject.title, from: "Subject"
-    select outcome.nickname, from: "Outcome"
+    selectize subject.title, from: "Subject"
+    selectize outcome.nickname, from: "Outcome"
     click_button t('helpers.submit.coverage.create')
 
     within("#matched_outcomes") do
       expect(page).to have_content(subject.title)
       expect(page).to have_content(outcome.nickname)
     end
   end
+
+  def selectize(item, from:)
```

Comment:
Perhaps using the selectize javascript api is a good idea. I foresee needing to use it to implement the nice form designs in the mockups. That said, I am not 100 % sure as I do not think that using selectize will necessarily make the code cleaner. From the documentation, it looks like you need to call quite a few functions. Maybe we can look at this together; I am curious to see what you had in mind if the selectize api is used.